core/src/main/scala/cromwell/core/callcaching/CallCachingMode.scala

```diff
@@ -13,6 +13,8 @@ sealed trait CallCachingMode {
 }
 
 case object CallCachingOff extends CallCachingMode {
+  override val readFromCache = false
+  override val writeToCache = false
```

Comment:
I had some issues testing when call caching was turned off. This gave me more confidence but I'm okay removing it too since the ejea spec looks so darn thorough